### PR TITLE
was added requirements.txt #5

### LINKED ISSUE #5: Add requirements.txt
Please add `requirements.txt` file and define all packages will be used for this repo. Don't forget to update README with the instructions about how to install packages from this file.

### PR DESCRIPTION
**Pull request by qaivanpleten**
there are only 4 packages, which I use. I will update requirements file
Closes #5

### CODE REVIEW

**Review by extsoft — CHANGES_REQUESTED** [review]
@qaivanpleten please add all used packages to requirements file (the packages were installed with `pip install`).

**Comment by qaivanpleten** [comment]
@extsoft 
I added all packages which I use. All which I remember)

**Comment by extsoft** [comment]
> @extsoft
I added all packages which I use. All which I remember)

@qaivanpleten 
The file is empty in PR - https://github.com/qaivanpleten/repspark_ios_tests/pull/11/files.

**Comment by extsoft** [comment]
And use `pip freeze` to check the packages.

**Comment by qaivanpleten** [comment]
вроде пофиксил, сделал коммит и пуш реквест с фиксами. чекни плиз

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+altgraph==0.10.2
```
Comment (extsoft) [inline requirements.txt]: There are a lot of transitive dependencies in this file. You need only top level dependencies. To get them, please install `pip install pipdeptree` and run `pipdeptree`. You will see the output like 
```bash
pipdeptree==0.10.1 # skip this and its tree as pipdeptree is a debug tool 
  - pip [required: >=6.0.0, installed: 7.1.2] 
pytest==3.2.0 # add to requirements.txt
  - py [required: >=1.4.33, installed: 1.4.34] # transitive = skip
  - setuptools [required: Any, installed: 18.2] # transitive = skip
selenium==3.4.3 # add to requirements.txt
```

@qaivanpleten, please leave only main dependencies.

**Review by extsoft — CHANGES_REQUESTED** [review]

**Comment by qaivanpleten** [comment]
сделал. стало меньше. но все равно 43 пакета. хз какие из них нужны. предлагаю обсудить лично

**Comment by extsoft** [comment]
@qaivanpleten 
> сделал. стало меньше. но все равно 43 пакета. хз какие из них нужны. предлагаю обсудить лично

Try to use [pipreqs](https://github.com/bndr/pipreqs).

**Comment by qaivanpleten** [comment]
@extsoft 
I believe now there are only used packages

**Review by extsoft — APPROVED** [review]
Great job!

**Pull request merged**